### ISSUE TITLE
Files have different status (Syncing / Out of sync) on my two devices

### ISSUE THREAD

**Issue by mlell** [syncthing/syncthing#7826]
I am running Syncthing on Android (1.18.0)  and on Arch Linux (1.17.1) to sync my smartphone's storage to my notebook. If I open Syncthing on Linux, I get the message "Syncing.... 99%" on the device page of my Smartphone. I can click on "Out of sync items", which lists some files which are suposedly yet to be synchronized. However, If I open Syncthing on Android, I get a red message on the sync'ed folder "Nicht synchronisiert" (don't know the exact english translation. If I look into the logs, I see that several files are not synchronized due to errors like this

```
2021-07-18 10:27:29 Puller (folder "Speicher" (t66yq-r9o4p), item "Music/Einzel/The King's Singers/This Is The King's Singers"): syncing: checking file to be replaced: given name "Music/Einzel/The King's Singers/This Is The King's Singers" differs from name in filesystem "Music/Einzel/The King's Singers/This Is the King's Singers"
```

I think this is confusing. Both sides should agree on whether synchronization is running or is stopped and needs user intervention.

**Comment by AudriusButkevicius** [comment]
This looks more like a support request than a clear bug or feature request. Please use the forum - https://forum.syncthing.net/ - for support, first by searching for any similar issues and then by posting a new one when required. The forum has a lot more eyes and friendly people on it than the issue tracker.

Thanks!

**Comment by mlell** [comment]
@AudriusButkevicius If you expect that users should refer to the forums when they have problems on a fairly standard sync case, ok. But I thing that this might hamper the adoption of Syncthing by normal users, which is why I filed this as a bug report. 

The expected behaviour is that I can see in the web interface whether the sync is progressing or whether it needs manual intervention. 

The actual behaviour is that I see "Syncing..." on the web interface, with no indication that I need to do something. The items just show as "Out of sync". Only when I open the smartphone app, I see the folder state "not synchronized" ("Nicht synchronisiert"). This is no useful information that I know. I installed Syncthing because the folder is not synchronized. I want to synchronize it. Only when I open the logs that are hidden in the web interface on android I see what the actual problem is (still without a hint on how to solve it but that is then issue #7827)

I have a lot of experience in Linux administration, Python and R, so I can only speculate what a non-technical user might do. But I suspect that he will just stop and uninstall Syncthing after seeing that the Sync progress is stalled with no message on the notebook. Therefore, maybe consider this as a UI bug if it fits to the standards of this project.
